.github/workflows/tests.yml

```
@@ -70,6 +70,13 @@ jobs:
         pip install --use-deprecated=legacy-resolver scikit-optimize
         pip install --use-deprecated=legacy-resolver cma
 
+    # TODO: Add the deselected tests back after fixing version constraint of sklearn. 
```

A nit, but please remove the trailing whitespace.
```suggestion
    # TODO: Add the deselected tests back after fixing version constraint of sklearn.
```

---

Sorry about that 🙇

---

No problem. Thank you for your swift action!